index.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+
+<html lang="en">
+<head>
```

Comment:
Add a tab space for the head and body and an empty line to separate head, boy and html.